app/brave_main_delegate.cc

```diff
@@ -86,6 +89,17 @@ const char kBraveOriginTrialsPublicKey[] =
 
 const char kDummyUrl[] = "https://no-thanks.invalid";
 
+std::string GetUpdateURLHost() {
```

Comment:
this should belong to the anonymous namespace, together we some other vars above

Reply:
done!

Reply:
resolved!